netbox/ipam/api/views.py

```diff
@@ -155,7 +155,7 @@ def available_prefixes(self, request, pk=None):
             return Response(serializer.data)
 
     @swagger_auto_schema(method='get', responses={200: serializers.AvailableIPSerializer(many=True)})
-    @swagger_auto_schema(method='post', responses={201: serializers.AvailableIPSerializer(many=False)},
+    @swagger_auto_schema(method='post', responses={201: serializers.AvailableIPSerializer(many=True)},
                          request_body=serializers.AvailableIPSerializer(many=False))
```

Comment:
The many=False is required due to the way the body schema generation works as it cannot handle a list serializer.

Reply:
Why does `available_ips()` require specifying `request_body` but `available_prefixes()` does not?

Reply:
available_prefixes already uses the prefix serializer for the body.  With available_ips, unless you specify AvailableIPSerializer, it defaults to WriteablePrefix.  We can do the Prefix serializer too, if we want to be explicit, but it is redundant here I think